src/js/player.js

```diff
@@ -374,6 +375,8 @@ class Player extends Component {
 
     this.el_ = this.createEl();
 
+    this.options_.sources = Player.getTagSettings(this.tag).sources || this.options_.sources;
```

Comment:
we might want to just update `getTagSettings to know about `vjs-source` as well, instead of this, though, this seems to work.

Reply:
This breaks some tests, so, removing and updating `getTagSettings`.